prac_06/guitar_test.py

```diff
@@ -0,0 +1,12 @@
+"""
+CP1404 - PRAC 06
+"""
+from prac_06.guitar import Guitar
+
+gibson = Guitar("Gibson L-5 CES", 1922, 16035.40)
+bass = Guitar("Bass L-5 CES", 1999, 134563.40)
+
+print(f"{gibson.name} get_age() - Expected 102. Got {gibson.get_age()}")
```

Comment:
I like the way you set up your testing to print a string, in one line. Defiantly efficient and readable.